pkgs/tools/misc/dt-shell-color-scripts/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, bash
+, makeWrapper
+}:
+  stdenv.mkDerivation {
+    pname = "dt-shell-color-scripts";
+    version = "1.0.0";
+    src = fetchFromGitLab {
+      # https://gitlab.com/dwt1/shell-color-scripts
+      owner = "dwt1";
+      repo = "shell-color-scripts";
+      sha256 = "sha256-bd3NBf99rCiADUKQb6fzCBDaKBmYaZHcO4qokm/39do=";
+      # have to use revesion because this package has no version tags so I am pinning the latest commit
```

Comment:
There is no need to apologize here. There are many packages on Nixpkgs that use Git hashes.